### DISCUSSION TITLE
Is there a way to not report beta version as available major version upgrade?

### DISCUSSION THREAD

**Discussion by jeremymeng** [pnpm/pnpm#4275] [Q&A]
We use rush and pnpm to manage our mono-repo.  We check the output of `pnpm install` to find out the available major version upgrades then use automation to create work items for each of those.

We found that at some point of time, pnpm reports that `fast-xml-parser` has a major version of v 4.0.0 available when we were using v3.  However, at that time, only 4.0.0-beta.* versions are published to npmjs.  A stable version hadn't been released yet.

Is there a way to only check for upgrades that are stable versions?

**Comment by zkochan** [comment]
In the installation summary, pnpm shows any version that is under the "latest" dist-tag. So probably `4.0.0-beta.*` was published as "latest" for `fast-xml-parser`.

There is no way to change this behavior. I am also not sure this reporting should be configurable. But we can make the behavior of `pnpm outdated` more configurable.
